crates/bevy_animation/src/lib.rs

```diff
@@ -482,6 +482,8 @@ pub struct ActiveAnimation {
     completions: u32,
     /// `true` if the animation was completed at least once this tick.
     just_completed: bool,
+    /// `None` is mean animation was never completed forever.
```

Comment:
```suggestion
    /// `None`  means the animation was will never complete.
```

I'm not sure if this is correct, but I couldn't understand your original comment.